tests/Gedmo/Tool/BaseTestCaseORM.php

```diff
@@ -187,27 +173,23 @@ protected function stopQueryLog($dumpOnlySql = false, $writeToLog = false)
 
     /**
      * Creates default mapping driver
-     *
-     * @return MappingDriver
      */
-    protected function getMetadataDriverImplementation()
+    protected function getMetadataDriverImplementation(): MappingDriver
     {
         return new AnnotationDriver($_ENV['annotation_reader']);
     }
 
     /**
      * Get a list of used fixture classes
      *
-     * @return array
+     * @return array<class-string>
```

Comment:
```suggestion
     * @phpstan-return list<class-string>
```

Reply:
Is it make sense to restrict this to phpstan specific doc types? Wouldn't be better to stick to generic ones?

Reply:
the thing is that `class-string` is not a generic one I think, looking that the usage in this library, it was always prefixed https://github.com/doctrine-extensions/DoctrineExtensions/search?q=class-string

Reply:
Make sense :+1: